UI/data/locale/en-US.ini

```diff
@@ -1034,7 +1034,7 @@ Basic.Settings.Output.Simple.Warn.Encoder="Warning: Recording with a software en
 Basic.Settings.Output.Simple.Warn.Lossless="Warning: Lossless quality generates tremendously large file sizes! Lossless quality can use upward of 7 gigabytes of disk space per minute at high resolutions and framerates. Lossless is not recommended for long recordings unless you have a very large amount of disk space available. Replay buffer is unavailable when using lossless quality."
 Basic.Settings.Output.Simple.Warn.Lossless.Msg="Are you sure you want to use lossless quality?"
 Basic.Settings.Output.Simple.Warn.Lossless.Title="Lossless quality warning!"
-Basic.Settings.Output.Simple.Encoder.Software="Software (x264)"
+Basic.Settings.Output.Simple.Encoder.Software.X264="Software (x264)"
```

Comment:
I don't know how it will affect translations but it will be really better to keep `Software` as the x264 variant to avoid having to re-translate the sentence.

Just add a variant for openH264 for the dependent PR.

Reply:
I would really prefer to fix this rather than leaving it like that.

Reply:
I agree with @tytan652 - it makes more sense to keep the scope of this PR to explicitly add a new encoder (OpenH264) and nothing else. It would probably also make it easier to merge (because it'd a pure addition that doesn't impact any other code).

Changing the language is unnecessary and I'd argue that the changes also confuse because they add specificity to the encoder name without any need for it (because there is no entry "Software (OpenH264)" that would necessitate that distinction).